src/index.tsx

```diff
@@ -50,14 +57,28 @@ type Props = {
   };
   isNavigable?: boolean;
   maxSlices?: number;
+  onDataSelect?: (event: SelectEvent) => void;
 };
 
 export default function ReactFrappeChart(props: Props) {
   const ref = React.useRef<HTMLDivElement>(null);
   const chart = React.useRef<any>(null);
+  const {onDataSelect} = props;
 
   React.useEffect(() => {
-    chart.current = new Chart(ref.current, { ...props });
+    chart.current = new Chart(
```

Comment:
Can you run this through Prettier to be consistent with the rest of the codebase? Sorry, I think I should have added a pre-commit hook! 😅

Reply:
np, I just run prettier